omod/src/main/resources/org/openmrs/module/fhir2/narratives/datatypes/ContactPointDt.html

```diff
@@ -11,7 +11,6 @@
     </th:block>
 </th:block>
 <th:block th:unless="${resource.system}">
-    <th:block th:if="${resource.system}" th:text="${resource.system}+':'"></th:block>
-    <a th:if="${resource.value}" th:href="'mailto:' + ${resource.value}" th:text="${resource.value}"></a>
+    <th:block th:if="${resource.value}" th:text="${resource.value}"></th:block>
```

Comment:
This is probably on the list of things we should do, but we should add `mailto:` links for emails and `tel:` links for telephone numbers, where we know that information.

Reply:
@ibacher The `mailto:` link is actually present in case of emails (above in the file). The one I removed in this commit was a correction. I had added `mailto:` link to all contact points where `system` value was `null`. 

Sure I'll add the `tel:` link too.